tests/integration/llm/prepare.py

```diff
@@ -380,6 +380,12 @@
         "option.max_rolling_batch_size": 4,
         "option.tensor_parallel_degree": 4,
         "load_on_devices": 0
+    },
+    "llama2-7b-chat-gptq": {
```

Comment:
you are now putting them in a CI, but not installing the necessary wheel for the test. How this will work?